test/itemlist_formaction.cpp

```diff
@@ -536,8 +503,71 @@ TEST_CASE("OP_EDITFLAGS arguments are added to an item's flags", "[itemlist_form
 
 }
 
-TEST_CASE("process_op(OP_SAVE)", "[itemlist_formaction]") {
-	//NOTIMPL
+TEST_CASE("OP_SAVE writes an article's attributes to the specified file", "[itemlist_formaction]") {
+	controller c;
+	newsbeuter::view v(&c);
```

Comment:
Good point, messed it up in the first test, then propagated to all the other ones. Will fix.